projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/schema/plugin.json

```diff
@@ -2,6 +2,20 @@
   "jupyter.lab.shortcuts": [],
   "title": "vdk-jupyterlab-extension",
   "description": "vdk-jupyterlab-extension settings.",
+  "jupyter.lab.menus": {
+    "main": [
+      {
+        "id": "jp-vdk-menu",
+        "label": "VDK",
```

Comment:
should the label not be vdk run? 
Can you include a screen shot of what this looks like in the menu now?

Reply:
You can see this https://github.com/vmware/versatile-data-kit/blob/main/specs/vep-994-jupyter-notebook-integration/ux.mov

Reply:
This is the label of the menu. Every vdk command we add will be an item submenu of this menu. So "Run job" is an item of the menu.